frontend/src/components/common/SimpleTable.tsx

```diff
@@ -104,6 +112,34 @@ export default function SimpleTable(props: SimpleTableProps) {
   // eslint-disable-next-line
   [data, currentData]);
 
+  React.useEffect(() => {
+    if (data) {
+      let applySort = undefined;
+      const columnAskingForSort = columns.find((col) => col.label === sortObj.label);
```

Comment:
Don't use the label as it may not be unique and thus not a good way for distinguishing between columns. I think it's better to use the column index, this also makes getting the column more efficient.